src/App.tsx

```diff
@@ -59,6 +61,11 @@ const App = () => {
     };
   }, []);
 
+  polyfill({
+    dragImageTranslateOverride: scrollBehaviourDragImageTranslateOverride,
+    forceApply: true,
```

Comment:
I can't see an issue on desktop but I'm still kinda scared of this. As discussed, it'd be good to understand what it does at a code level on a non-touch-event device.

Reply:
As far as I understand, we're overwriting the `touchstart` event. Without the `forceApply` option, it will detect if dnd is natively supported, and only overwrite the `touchstart` event if it's not. See https://github.com/timruffles/mobile-drag-drop/blob/master/src/index.ts.

We can also add a `holdToDrag` option in ms if you think this is useful?